app/_gateway_entities/license.md

```diff
@@ -4,16 +4,12 @@ content_type: reference
 entities:
   - license
 
-description: A {{site.base_gateway}} license object lets you manage Enterprise licenses.
+description: A {{site.base_gateway}} License entity allows you manage Enterprise licenses.
```

Comment:
For reviewers: I struggled with "License" vs "license". I left it as "license" when it was an adjective ex. license report. If you could just double check the caps vs no caps, that would be great.